### DISCUSSION TITLE
Pinned Apps - "macOS Style"

### DISCUSSION THREAD

**Discussion by thomasza92** [lawand-dot-io/taskbar#79] [Feature requests]
Wasn't sure how to title this, but it would be cool if we could set some pinned apps (like Finder or Settings) to function how they normally would on macOS (so they do not open a task window on the taskbar). This way, I could have specific pinned apps like Finder and Settings open, yet have them remain as just small icons on the bottom left. It would also be nice if pinned apps could be configured so they could be aligned to either the left or the right of the taskbar.

I hope this is a sufficient explanation. Great work!

**Comment by lawand-dot-io** [comment]
Hello. I might understand what you mean. Can you try opening Settings then pinning it using the "Keep in Taskbar" option and then right clicking it and choosing "Add to Blacklist"? Does this achieve the behavior you meant?

**Reply by thomasza92** [reply]
Yes, it does! Thanks!
